scripts/backup-restore.sh

```
@@ -29,8 +30,8 @@ fi
 
 echo "For which instance do you want to import the backup?"
 read -r org
-folder=c-$org
-cd /var/docker/$folder
+folder="${PREFIX}$org"
+cd "$baseDirectory/$folder"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)